.circleci/config.yml

```diff
@@ -285,31 +285,46 @@ commands:
 # -------------------------
 jobs:
   api-build-test:
+    parallelism: 6
     # We're hitting OOM failures as of 1 May 2020.  Increase to 6GB RAM by specifying a medium+ machine.
     # Refers to https://circleci.com/docs/2.0/configuration-reference/#resource_class
     resource_class: medium+
     <<: *java_defaults
     steps:
       - checkout_init_git
-      - ensure_branch_has_changes:
-          dir_name: "api"
+      #- ensure_branch_has_changes:
+          #dir_name: "api"
       - manage_api_cache:
           restore: true
       - run:
           name: Validate Swagger definitions
           working_directory: ~/workbench/api
-          command: ./project.rb validate-swagger --project-prop verboseTestLogging=yes
+          command: |
+            if [ "$CIRCLE_NODE_INDEX" == 0 ]; then
+              ./project.rb validate-swagger --project-prop verboseTestLogging=yes
+            fi
       - run:
           name: Run Java unit tests
           working_directory: ~/workbench/api
-          command: ./project.rb test
+          command: |
+            CLASSNAMES=$(circleci tests glob "src/test/java/org/pmiops/workbench/**/*.java" \
```

Comment:
I would pull everything under `src/test/java`. If we added another package there, we'd still want those here I think.

Also, some comment or description attribute here would help. I'm not p on my `sed`isms.